modules/search/instant-search/components/search-widget.jsx

```diff
@@ -16,7 +16,7 @@ import debounce from 'lodash/debounce';
 import SearchResults from './search-results';
 import SearchFiltersWidget from './search-filters-widget';
 import { search, buildFilterAggregations } from '../lib/api';
-import { setSearchQuery } from '../lib/query-string';
+import { setSearchQuery, setFilterQuery, getFilterQuery } from '../lib/query-string';
```

Comment:
Just a drive by-comment, but to save on the bundle size, it might make sense to rely on external [`@wordpress/url`](https://github.com/WordPress/gutenberg/tree/a491a79ab2ec9c05b296c50e1ce0c5e389e19175/packages/url#readme)